core/server/data/migrations/versions/4.0/26-add-cascade-on-delete.js

```diff
@@ -0,0 +1,75 @@
+const logging = require('../../../../../shared/logging');
+const {createIrreversibleMigration} = require('../../utils');
+const {addForeign, dropForeign} = require('../../../schema/commands');
+
+module.exports = createIrreversibleMigration(async (knex) => {
+    if (knex.client.config.client !== 'sqlite3') {
+        return logging.warn('Skipping adding "on delete cascade" - database is not SQLite3');
+    }
+
+    logging.info('Adding on delete cascade for members_labels, members_stripe_customers and members_stripe_customers_subscriptions');
+
+    await dropForeign({
+        fromTable: 'members_labels',
+        fromColumn: 'member_id',
+        toTable: 'members',
+        toColumn: 'id',
+        transaction: knex
+    });
+    await addForeign({
+        fromTable: 'members_labels',
+        fromColumn: 'member_id',
+        toTable: 'members',
+        toColumn: 'id',
+        cascade: true,
```

Comment:
Not a big deal, but considering `CASCADE` can be used in multiple contexts, it might make sense to have this option named `cascadeDelete` like in the schema. Doesn't matter though if you cba to change it

Reply:
just learned "cba" 😂  also I've changed it, thanks!